TASVideos/Pages/Submissions/View.cshtml

```diff
@@ -37,6 +37,17 @@
 				return "warning";
 		}
 	}
+	string GetDisplayName(string propertyName) {
```

Comment:
This needs to not do the reflection piece every time the page is loaded. Better is to put this in the C# so it can be "cached" with a static value